project/frontend/src/components/MachineInfo.js

```diff
@@ -103,9 +111,10 @@ function MachineInfo({ auth }) {
     }
     function returnConsoleImage(console_name) {
         switch (console_name) {
-            case "PS5": return ps5_logo;
+            case "PS4": return ps5_logo;
```

Comment:
@AlexCHLo Why is it called ps4 and ps5 at the same time? Otherwise the changes look good

Reply:
Thanks for double-checking.  Really don't have a good reason it's like that, probably an old coding name that I didn't change. I'll update the code where it's more readable.